treq/testing.py

```diff
@@ -20,26 +20,12 @@
 from twisted.web.server import Site
 from twisted.web.iweb import IAgent, IBodyProducer
 
-from twisted.python.failure import Failure
-
 from zope.interface import directlyProvides, implementer
 
 import treq
 from treq.client import HTTPClient
 
 
-class AbortableStringTransport(StringTransport):
```

Comment:
Technically this is getting rid of public API I think, but maybe we can afford to be a little more fast and loose than Twisted itself.  (We should really make all implementation modules private by default, and explicitly populate public APIs …)